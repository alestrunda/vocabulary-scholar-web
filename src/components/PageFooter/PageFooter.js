import React from 'react';
import { APP_VERSION } from '../../constants';

const PageFooter = props => {
  return (
    <footer className="page-footer" {...props}>
      <div className="container">v{APP_VERSION}</div>
    </footer>
  );
};

export default PageFooter;
