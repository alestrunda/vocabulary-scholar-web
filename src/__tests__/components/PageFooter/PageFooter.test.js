import React from 'react';
import { shallow } from 'enzyme';

import PageFooter from '../../../components/PageFooter';

it('renders correctly', () => {
  const root = shallow(<PageFooter />);
  expect(root).toMatchSnapshot();
});
