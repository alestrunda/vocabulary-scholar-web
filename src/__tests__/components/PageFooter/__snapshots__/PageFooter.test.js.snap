// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PageFooter />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container"
      >
        v
        1.0.0
      </div>,
      "className": "page-footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "v",
          "1.0.0",
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        "v",
        "1.0.0",
      ],
      "type": "div",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container"
        >
          v
          1.0.0
        </div>,
        "className": "page-footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "v",
            "1.0.0",
          ],
          "className": "container",
        },
        "ref": null,
        "rendered": Array [
          "v",
          "1.0.0",
        ],
        "type": "div",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
